src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -205,3 +235,73 @@ const getPublicness = (currentTabName: typeof TABS[number]) => {
       return undefined
   }
 }
+
+export const useDeleteVideo = (memberId: string | null) => {
+  const { joystream } = useJoystream()
+  const [videoIdToDelete, setVideoIdToDelete] = useState<undefined | string>()
```

Comment:
I think it would be better to go for `null` here instead, so we don't need to pass those `undefined` below. For videoId we could go with `videoId ?? null`